webapp/components/wpt-results.html

```diff
@@ -343,7 +343,7 @@ <h3>
           },
           diffURL: {
             type: String,
-            computed: 'computeDiffURL(testRuns, query)',
+            computed: 'computeDiffURL(testRuns)',
```

Comment:
Are changes in this file a bug fix? Change from `this.diffFilter` to `this.filter` without corresponding property changes seems non-trivial.